system/jlib/jfile.cpp

```diff
@@ -1996,11 +1996,20 @@ void CFileIO::flush()
 
 offset_t CFileIO::size()
 {
+#ifndef _WIN32
+    //MORE: The current implementation of openHandle() calls fstat() to check the file is not a directory
+    //If the file hasn't been modified since open() the size could be cached and returned.
+    struct stat info;
+    if (fstat(file, &info) >= 0)
```

Comment:
Is fstat() info such as size 100% reliable over NFS (HPC-Cache?) to recent file changes ?